vulnerabilities/scraper/lwn.py

```diff
@@ -0,0 +1,118 @@
+# Author: Navonil Das (@NavonilDas)
```

Comment:
Consider removing this, or put your name instead :))

Reply:
Done